mappings/net/minecraft/network/PacketByteBuf.mapping

```diff
@@ -835,6 +835,24 @@ CLASS net/minecraft/class_2540 net/minecraft/network/PacketByteBuf
 	METHOD method_37453 getMaxValidator (Ljava/util/function/IntFunction;I)Ljava/util/function/IntFunction;
 		ARG 0 applier
 		ARG 1 max
+	METHOD method_42064 readRegistryValue (Lnet/minecraft/class_2359;)Ljava/lang/Object;
```

Comment:
Mind update that giant table in the class javadoc as well?

Also would be nice to note that callers are to ensure the registries/indexed iterables are matched on two sides before using these

Reply:
Ah yeah, fixing